### ISSUE TITLE
Inline image accessibility not announced

### ISSUE THREAD

**Issue by xmflsct** [tooot-app/app#96]

**Comment by xmflsct** [comment]
Same for inline links, hashtags, mentions.

**Comment by xmflsct** [comment]
https://github.com/facebook/react-native/issues/31298
